src/DepthImageToLaserScanROS.cpp

```diff
@@ -27,65 +27,88 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-/* 
+/*
  * Author: Chad Rockey
  */
 
+#include <rclcpp/rclcpp.hpp>
+#include <sensor_msgs/msg/camera_info.hpp>
+#include <sensor_msgs/msg/image.hpp>
+#include <sensor_msgs/msg/laser_scan.hpp>
+
+#include <functional>
+
 #include <depthimage_to_laserscan/DepthImageToLaserScanROS.h>
 
+#define ROS_ERROR printf
```

Comment:
That's fine with me.  I was having trouble making it work right now anyway.  Once we get that more fully implemented, and we are past beta2, I'll make a sweep back through the code and fix a lot of these.